components/common-go/baseserver/server.go

```diff
@@ -7,6 +7,7 @@ package baseserver
 import (
 	"context"
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus/collectors"
```

Comment:
It'd be good if we could couple the import package `"github.com/prometheus/client_golang` together.
```go
        "github.com/prometheus/client_golang/prometheus"
        "github.com/prometheus/client_golang/prometheus/collectors"
	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Reply:
If we do want this behaviour, we should ge a linter to automatically re-order the imports as this is purely cosmetic but creates noise and toil in dev cycle. I'll fix it here for now.